projects/sbb-esta/angular-business/dialog/src/dialog/dialog-container/dialog-container.component.scss

```diff
@@ -8,7 +8,7 @@
 sbb-dialog-container {
   @include dialogContainer();
 
-  & > * {
+  :only-child {
```

Comment:
Good find 😃

Reply:
Too bad it didn't hold up because of `ViewEncapsulation.NONE` :(